src/bn256/fq.rs

```diff
@@ -197,7 +204,12 @@ impl ff::PrimeField for Fq {
 
     const NUM_BITS: u32 = 254;
     const CAPACITY: u32 = 253;
-
+    const MODULUS: &'static str = MODULUS_STR;
+    const MULTIPLICATIVE_GENERATOR: Self = MULTIPLICATIVE_GENERATOR;
+    const ROOT_OF_UNITY: Self = ROOT_OF_UNITY;
+    const ROOT_OF_UNITY_INV: Self = ROOT_OF_UNITY_INV;
+    const TWO_INV: Self = TWO_INV;
+    const DELTA: Self = DELTA;
     const S: u32 = 0;
```

Comment:
I think it would be worth it computing it. 
Same happens with `ROOT_OF_UNITY` for `Fq` and `Fq2`. I think we don't loose anything by putting the correct values even if aren't used. 

I think @kilic can provide input on this part as he was the OG implementor of the Bn curve in this crate.

Reply:
For bn256 `ROOT_OF_UNITY` for `Fq` and `Fq2` are left blank intentially since those fields has nothing to do with fft. But yes we can update them with correct values